### ISSUE TITLE
Why brouter routes through cycleway with acccess=no

### ISSUE THREAD

**Issue by zzptichka** [abrensch/brouter#208]
This particular case: http://brouter.de/brouter-web/#map=17/45.40321/-75.71102/standard&lonlats=-75.71197,45.401851;-75.714172,45.404143
Due to construction `access=no` tag has been there since June

**Comment by Phyks** [comment]
This cycleway is part of a cycle relation (https://www.openstreetmap.org/relation/7175029#map=15/45.4035/-75.7163) and BRouter ignores access tags for ways part of a cycle relation. See https://github.com/nrenner/brouter-web/issues/31

**Comment by zzptichka** [comment]
Ah I see. I guess I'll tag it as `highway=construction` then.

**Comment by zzptichka** [comment]
It appears it routes through `highway=construction` as well. Hmm.
So is the only solution to remove it from the relation?

**Comment by Phyks** [comment]
Indeed, the reason being that BRouter can routes over `highway=construction`, see https://github.com/abrensch/brouter/issues/62.

So:
* BRouter avoids `access=no` unless the way is part of a cycle route.
* BRouter considers `highway=construction` as routable.

Concerning the second point, the main reason is that usually when a highway is under construction, pedestrian (and bicycles) can still pass on the side. Prolem here is that it is a cycleway which is under construction, not a regular highway.

Maybe BRouter should take into acount the value of the `construction=` tag to decide whether to route on it or not.

**Comment by m4lvin** [comment]
On a ride today I had this issue, see [here on bikerouter.de](https://brouter.m11n.de/#map=17/51.41883/7.36864/standard&lonlats=7.366281,51.418212;7.371826,51.417783) vs the same stretch of road [on openstreetmap.org with OSRM](https://www.openstreetmap.org/directions?engine=fossgis_osrm_bicycle&route=51.41786%2C7.36739%3B51.417519%2C7.370946#map=17/51.419861/7.369648). I actually thought this must be mapped wrong when I arrived at the private and blocked road and wanted to correct the map data when I got home. Turns out most brouter profiles happily use this road even though it has `access=no`.

**Comment by mcliquid** [comment]
As far as I know, the bicycle relation overruled the access tag. If access=no applies here, then the relation is also wrong here IMO.
